frontend/src/libs/helpers/get-reaction-converted-to-boolean/get-reaction-converted-to-boolean.helper.ts

```diff
@@ -0,0 +1,10 @@
+import { Reaction } from '~/libs/enums/enums.js';
+import { type ValueOf } from '~/libs/types/types.js';
+
+const getReactionConvertedToBoolean = (
```

Comment:
Not sure if it should be common helper - as it's used only in one place for article